package.json

```diff
@@ -66,8 +66,8 @@
     "@emotion/react": "^11.13.3",
     "@eslint/eslintrc": "^3.1.0",
     "@eslint/js": "^9.11.1",
-    "@instructure/browserslist-config-instui": "10.2.2",
-    "@instructure/pkg-utils": "10.2.2",
+    "@instructure/browserslist-config-instui": "10.3.0",
+    "@instructure/pkg-utils": "10.3.0",
```

Comment:
These dont update automatically for some reason on `npm run bump`, can you please create a ticket for this?